api/resources_portal/views/document_views.py

```diff
@@ -47,6 +47,8 @@ class MaterialDocumentSerializer(serializers.Serializer):
     organization = serializers.SerializerMethodField(read_only=True)
     mta_attachment = serializers.SerializerMethodField(read_only=True)
     additional_metadata = serializers.SerializerMethodField(read_only=True)
+    imported = serializers.BooleanField(read_only=True)
```

Comment:
This should also be added to the class meta on line 66.

Reply:
good catch!